UIDokan/Sources/UIDokan/Style/StackViews/LabelAndTextfield.swift

```diff
@@ -0,0 +1,38 @@
+//
```

Comment:
- StackView should be moved under `UIDokan/Sources/ReusableViews`
- Add each view with nib inside a separate group